homeassistant/helpers/config_validation.py

```diff
@@ -446,11 +446,19 @@ def time_period_str(value: str) -> timedelta:
     elif value.startswith("+"):
         value = value[1:]
 
+    days = ""
```

Comment:
days should not be initialized as a string, instead, make it an int from the get-go.